test/update-db.test.js

```diff
@@ -60,126 +97,105 @@ it('throws on missing lockfile', async () => {
   )
 })
 
-it('updates caniuse-lite for npm', async () => {
-  let dir = await chdir('update-npm', 'package.json', 'package-lock.json')
+it('shows target browser changes', async () => {
```

Comment:
moved this up - it is unchanged apart from being in a different position in the file